apps/namadillo/src/App/AccountOverview/AssetsOverviewPanel.tsx

```diff
@@ -0,0 +1,37 @@
+import { TabContainer } from "@namada/components";
+import clsx from "clsx";
+import { useRequiresNewShieldedSync } from "hooks/useRequiresNewShieldedSync";
+import { useState } from "react";
+import { ShieldedAssetsOverview } from "./ShieldedAssetsOverview";
+import { UnshieldedAssetsOverview } from "./UnshieldedAssetsOverview";
+
+export const AssetsOverviewPanel = (): JSX.Element => {
+  const requiresNewShieldedSync = useRequiresNewShieldedSync();
+
+  // Always select the unshielded tab as default if shielded sync is required
+  const [currentTab, setCurrentTab] = useState(requiresNewShieldedSync ? 1 : 0);
```

Comment:
@pedrorezende what do you think about using the `useUrlState` here?

Reply:
I dont think `useUrlState` is going to even accomplish what we want here unfortunately. 

B/c basically everytime you update the route, by clicking on any buttons, the state is lost.